source/localizable/tutorial/routes-and-templates.md

```diff
@@ -154,7 +154,7 @@ Here we will use the `{{link-to}}` helper in our code to link between routes:
 ```
 
 The `{{link-to}}` helper takes an argument with the name of the route to link to, in this case: `contact`.
-When we look at our about page, we now have a working link to our contact page.
+When we look at our about page at `http://localhost:4200/about`, we now have a working link to our contact page.
```

Comment:
Should we make that link clickable?  I kind of like that idea, but it might not be a good idea ... ;-)